src/common/codeine/AbstractCodeineBootstrap.java

```diff
@@ -131,6 +139,11 @@ private void registerPrometheus() {
         new JettyStatisticsCollector(stats).register();
     }
 
+    private void addHealthEndpoint(ServletContextHandler handler, HealthCheckRegistry healthCheckRegistry) {
+        log.info("Adding health end point");
+//        handler.addServlet(new ServletHolder(new HealthServlet(healthCheckRegistry)), Constants.HEALTH_CONTEXT);
```

Comment:
?

Reply:
yes, this not needed.. forgot to delete it